plugins/package-curation-providers/clearly-defined/src/funTest/kotlin/ClearlyDefinedPackageCurationProviderFunTest.kt

```diff
@@ -37,26 +40,33 @@ class ClearlyDefinedPackageCurationProviderFunTest : WordSpec({
         "return an existing curation for the javax.servlet-api Maven package" {
             val packages = createPackagesFromIds("Maven:javax.servlet:javax.servlet-api:3.1.0")
 
-            val curations = provider.getCurationsFor(packages)
+            withRetry {
```

Comment:
I'd prefer to only wrap the actual `getCurationsFor` call in `withRetry`, like

    val curations = withRetry { provider.getCurationsFor(packages) }

Reply:
IIUC it doesn't work, because `provider.getCurationsFor(packages)` sometimes returns empty set unexpectedly.

Reply:
You're right.